<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit_computer</class>
 <widget class="QDialog" name="Edit_computer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_name">
         <property name="text">
          <string>&lt;span style='color:#ff3366'&gt;*&lt;/span&gt; Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="name_edit">
         <property name="placeholderText">
          <string>Enter name...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_year">
         <property name="text">
          <string>Build year:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="year_edit">
         <property name="displayFormat">
          <string>yyyy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_type">
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_type">
         <item>
          <property name="text">
           <string>Mechanical</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Transistor</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Electronic</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="check_was_built">
         <property name="text">
          <string>was built</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="save_button">
         <property name="text">
          <string>Save changes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancel_button">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
